schema/types/base/sortedAccAddresses.go

```diff
@@ -64,6 +64,7 @@ func (accAddresses sortedAccAddresses) Delete(i interface{}) types.SortedList {
 }
 func (accAddresses sortedAccAddresses) Search(i interface{}) int {
 	accAddress := accAddressFromInterface(i)
+
```

Comment:
revert

Reply:
This was giving a lint error when I tried to push. " return statements should not be cuddled if block has more than two lines (wsl) "

Reply:
ahh, then makes sense